Dockerfile

```diff
@@ -1,9 +1,9 @@
-ARG APP_INSIGHTS_AGENT_VERSION=2.3.1
-FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.0
-
-EXPOSE 8080
+ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
```

Comment:
java logging needs updating as well
https://github.com/hmcts/ccpay-payment-app/blob/master/api/build.gradle#L50

`java-logging-spring` should be removed, and the other libs need to drop 'java' from the front

Reply:
done